src/cascadia/TerminalApp/SettingsTab.cpp

```diff
@@ -23,9 +23,11 @@ namespace winrt
 
 namespace winrt::TerminalApp::implementation
 {
-    SettingsTab::SettingsTab(MainPage settingsUI)
+    SettingsTab::SettingsTab(MainPage settingsUI,
+                             winrt::Windows::UI::Xaml::ElementTheme requestedTheme)
     {
         Content(settingsUI);
+        _requestedTheme = requestedTheme;

```

Comment:
Why do we need `requestedTheme` in the ctor? Can't we just get it from `settings.GlobalSettings().CurrentTheme().RequestedTheme()` like in `UpdateSettings()` below?

Reply:
Oh... we don't have settings... just the settings UI... there's no way to get the settings model object? I'd believe it, just surprised since we're literally a settings tab hahaha

Reply:
I know it's disgusting